test/integration/helpers/testHelpers.ts

```diff
@@ -0,0 +1,45 @@
+import { readFileSync } from 'fs';
+import { DetailedPopulationGroupResult, ExecutionResult, PopulationResult } from '../../../src/types/Calculator';
+import { PopulationType } from '../../../src/types/Enums';
+
+/**
+ * Retrieves JSON fixture from the integration directory for easier ts casting in test files
+ */
+export function getJSONFixture(path: string): any {
+  return JSON.parse(readFileSync(`test/integration/${path}`).toString());
+}
+
+/**
+ * Parses through the execution result and asserts that the population group at the specified index exists.
+ * Returns that group if so
+ */
+export function getGroupByIndex(
```

Comment:
How about changing this to be by groupId, not by index? Every result should have a group ID

Reply:
I thought group id was not required, at least in base FHIR.

Reply:
It's not, but I believe fqm-execution puts a default there when not provided to be `group-${index}`

See https://github.com/projecttacoma/fqm-execution/blob/90b1951e7ba5b8a41c9b93e3c873cae243496134/src/types/Calculator.ts#L111 -- groupId is always there

Reply:
The reason I didn't do this is that regenerating the bundle will cause all the groupIds to be regenerated (right?) meaning every test will need to be updated any time we change/add to the CQL. Is this a concern? @mgramigna @hossenlopp

Reply:
OH you're 100% right lol